src/get_target_data.py

```diff
@@ -427,7 +430,8 @@ def write_target_data(
             ("location", pa.string()),
             ("target_date", pa.date32()),
             ("clade", pa.string()),
-            ("oracle_value", pa.float64()),
+            ("oracle_value", pa.int64()),
```

Comment:
In this case, `oracle_value` represents a count of sequences, so int is a more appropriate data type.